src/script/interpreter.h

```diff
@@ -144,7 +148,7 @@ uint256 SignatureHash(const CScript& scriptCode, const T& txTo, unsigned int nIn
 class BaseSignatureChecker
 {
 public:
-    virtual bool CheckSig(const std::vector<unsigned char>& scriptSig, const std::vector<unsigned char>& vchPubKey, const CScript& scriptCode, SigVersion sigversion) const
+    virtual bool CheckSig(const std::vector<unsigned char>& scriptSig, const std::vector<unsigned char>& vchPubKey, const CScript& scriptCode, SigVersion sigversion, const bool require_secure_single) const
```

Comment:
Should `require_secure_single` be an unnamed parameter since it is not used?

Reply:
Just for consistency?

Reply:
Oh, sorry. Commented in the wrong file. I meant to comment in the `DummySignatureChecker` below.